general/design-dev-cycle.md

```diff
@@ -0,0 +1,19 @@
+Product/Design development cycle for source{d} application:
+
+- a) Team Sport
+- b) Problem Discovery
+- c) Ideation & Validation (low-fi interactive draft presentation & discussion)
+- d) Wireframing (scoped features interactive design asset)
+- e) Wireframes User Testing (scripted & recorded proto-usabllity tests)
+- f) Research Analysis & Conclusions
+- g) Wireframes Iteration (asset ready for implementation)
+- h) Development Roadmap/Planning & Metrics Assessment Definition
```

Comment:
If I understood it, I'd clarify which team leads each step.
If I understood it, it could be &ndash;roughly&ndash;
- `a` &rarr; `g`: product/design
- `h` &rarr; `j`: engineering
- `k` &rarr; `m`: product
- `n` &rarr; `o`: engineering
- `p`: engineering